jpegoptim.manifest

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Is this really needed? Isn't a default manifest embedded?

BTW this file should have a missing EOF line as the other newly added files.

Reply:
An EOF line has been added.
The default manifest (which appears in your existing builds) is:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
    <security>
      <requestedPrivileges>
        <requestedExecutionLevel level="asInvoker" uiAccess="false"></requestedExecutionLevel>
      </requestedPrivileges>
    </security>
  </trustInfo>
</assembly>
```
Here this is merged with the following manifest to [enable the UTF-8 codepage](https://learn.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page) to provide working Unicode path support on Windows without any C source code modifications:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
  <application>
    <windowsSettings>
      <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
    </windowsSettings>
  </application>
</assembly>
```

Without this manifest, attempting to optimize a file with a Unicode path on Windows results in this behaviour:
```
> jpegoptim ✨.jpg
jpegoptim: file not found: ?.jpg
```
With it, you will instead see this:
```
> jpegoptim✨.jpg
Γ£¿.jpg 2304x3072 24bit N JFIF  [OK] 1567683 --> 1475048 bytes (5.91%), optimized.
```
Which isn't perfect, but highly functional. It works on Windows Version 1903 (May 2019 Update) and up, and will be ignored without any side effects on older systems.

Reply:
@Eta0 Unfortunately, this broke unicode characters for me; previously it worked on my system which has a Greek locale, but now any jpg which is in a path with Greek characters is skipped. Going back to my latest fork build things work.